src/main/java/com/capgemini/setrack/model/Room.java

```diff
@@ -3,21 +3,41 @@
 import com.capgemini.setrack.model.enums.RoomStatus;
 
 import javax.persistence.*;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
 
 @Entity
-public class Room {
+public class Room extends Model {
     @Id
     @GeneratedValue(strategy = GenerationType.AUTO)
     private long id;
 
     @ManyToOne
     @JoinColumn(name="room_type_id")
+    @NotNull(message="A room type is required!")
```

Comment:
Onnodig, doordat het een dropdown menu is, hoor je niet niks te kunnen selecteren

Reply:
Houden we er toch in omdat het om server side validatie gaat